File: zorex-salvo/maria-health-phi/fibonacci.js
const count = 10;
let prev = 0;
let next = 1;
let current;

for (var i = 0; i < count; i++) {
  if (prev % 3 === 0) {
    console.log('Maria');
  } else if (prev % 5 === 0) {
    console.log('Health');
  } else if (prev % 5 === 0 && prev % 3 === 0) {
    console.log('Maria Health');
  } else {
    console.log(prev);
  }
  current = prev + next;
  prev = next;
  next = current;
}

File: zorex-salvo/maria-health-phi/fibonacci.go
package main
import "fmt"

func main() {
	count := 10
	prev := 0
	next := 1
	current := 0

	for i := 0; i < count; i++ {
		if (prev % 3 == 0) {
			fmt.Println("Maria")
		} else if (prev % 5 == 0) {
			fmt.Println("Health")
		} else if (prev % 3 == 0 && prev % 5 == 0) {
			fmt.Println("Maria Health")
		} else {
			fmt.Println(prev)
		}
		
		current = prev + next
		prev = next
		next = current
		
	}

}
